### ISSUE TITLE
Problem running the create_statistics.sh script

### ISSUE THREAD

**Issue by johann-petrak** [rsennrich/ParZu#24]
It is not possible for me to run `./statistics/create_statistics.sh conllfile` because then the prolog files are not found. 

So I cd into statistics and run `./create_statistics.sh ../conllfile` 

The first problem is in stats_creator/conll2prolog.py line 70 which uses a python 2 print statement which will not work with Python 3. This can be fixed by changing the line into
```python
print("w({0}, {1}, '{2}', '{3}', '{4}', {5}, {6}).".format(i,pos,token,tag,label,head,morph))
```
for Python 3.

The second problem is that there is no guard for conll input lines which do not have at
least 11 fields. partA of my converted hamburg dependency treebank has only 10 fields in 
line  1378186 so the python program crashes there. This can be fixed by replacing the line 
with the split function by:
```python
    fields = line.strip().split()
    if len(fields) < 11:
        print("Line {} has only {} fields, ignored".format(linenr, len(fields)), file=sys.stderr)
        continue
    pos, word, lemma, _, tag, _, morph, _, head, _, label = fields[:11]
```
(and also adding the linenr counter).

**Comment by rsennrich** [comment]
thanks; this was a bit more involved, since the position of features, and the formatting of the morphological features, is also different. The script should now support both current versions of TüBa-D/Z and HDT.

**Comment by johann-petrak** [comment]
Thanks!

When I try this, it now looks as if the  create_statistics.sh script can directly digest the original CONLL-format version of the HDT while what I did was to use the statistics/stats_creator/hamburg_treebank_to_conll.py script first to convert the HDT to Conll, and *that* file then created the problem with the incorrect number of fields. 

However, when I tried now to run the create_statistics.sh script with the output of the hamburg to conll script, I got an error 
```
ERROR: '<meta-call>'/1: Undefined procedure: w/7
   Exception: (11) w(_2524, _2530, _2558, ne, _2562, _2536, _2542) ? 
```

This is not really a problem since using the original HDT conll file is probably better anyways, but it may be confusing because the README.md file still suggests that this should work, if I understand correctly.
